packages/web-components/src/tabs/tabs.stories.ts

```diff
@@ -0,0 +1,235 @@
+  <fluent-tabs size="medium">
+    <fluent-tab id=${defaultTabIds[0]}> First Tab </fluent-tab>
+    <fluent-tab id=${defaultTabIds[1]}> Second Tab</fluent-tab>
+    <fluent-tab id=${defaultTabIds[2]}> Third Tab</fluent-tab>
+    <fluent-tab id=${defaultTabIds[3]}> Fourth Tab</fluent-tab>
+
+    <fluent-tab-panel id=${defaultTabIds[0] + '-panel'}></fluent-tab-panel>
+    <fluent-tab-panel id=${defaultTabIds[1] + '-panel'}></fluent-tab-panel>
+    <fluent-tab-panel id=${defaultTabIds[2] + '-panel'}></fluent-tab-panel>
+    <fluent-tab-panel id=${defaultTabIds[3] + '-panel'}></fluent-tab-panel>
+  </fluent-tabs>
+  <fluent-tabs size="medium" orientation="vertical">
+    <fluent-tab id=${defaultTabIds[0]}> First Tab </fluent-tab>
+    <fluent-tab id=${defaultTabIds[1]}> Second Tab</fluent-tab>
+    <fluent-tab id=${defaultTabIds[2]}> Third Tab</fluent-tab>
+    <fluent-tab id=${defaultTabIds[3]}> Fourth Tab</fluent-tab>
+
+    <fluent-tab-panel id=${defaultTabIds[0] + '-panel'}></fluent-tab-panel>
+    <fluent-tab-panel id=${defaultTabIds[1] + '-panel'}></fluent-tab-panel>
+    <fluent-tab-panel id=${defaultTabIds[2] + '-panel'}></fluent-tab-panel>
+    <fluent-tab-panel id=${defaultTabIds[3] + '-panel'}></fluent-tab-panel>
+  </fluent-tabs>
+`;
+export const TabsSizeMedium = renderComponent(tabsSizeMedium).bind({});
+
+const tabsSizeLarge = html`
+  <fluent-tabs size="large">
+    <fluent-tab id=${defaultTabIds[0]}> First Tab </fluent-tab>
+    <fluent-tab id=${defaultTabIds[1]}> Second Tab</fluent-tab>
+    <fluent-tab id=${defaultTabIds[2]}> Third Tab</fluent-tab>
+    <fluent-tab id=${defaultTabIds[3]}> Fourth Tab</fluent-tab>
+
+    <fluent-tab-panel id=${defaultTabIds[0] + '-panel'}></fluent-tab-panel>
+    <fluent-tab-panel id=${defaultTabIds[1] + '-panel'}></fluent-tab-panel>
+    <fluent-tab-panel id=${defaultTabIds[2] + '-panel'}></fluent-tab-panel>
+    <fluent-tab-panel id=${defaultTabIds[3] + '-panel'}></fluent-tab-panel>
+  </fluent-tabs>
+  <fluent-tabs size="large" orientation="vertical">
+    <fluent-tab id=${defaultTabIds[0]}> First Tab </fluent-tab>
+    <fluent-tab id=${defaultTabIds[1]}> Second Tab</fluent-tab>
+    <fluent-tab id=${defaultTabIds[2]}> Third Tab</fluent-tab>
+    <fluent-tab id=${defaultTabIds[3]}> Fourth Tab</fluent-tab>
+
+    <fluent-tab-panel id=${defaultTabIds[0] + '-panel'}></fluent-tab-panel>
+    <fluent-tab-panel id=${defaultTabIds[1] + '-panel'}></fluent-tab-panel>
+    <fluent-tab-panel id=${defaultTabIds[2] + '-panel'}></fluent-tab-panel>
+    <fluent-tab-panel id=${defaultTabIds[3] + '-panel'}></fluent-tab-panel>
+  </fluent-tabs>
+`;
+export const TabsSizeLarge = renderComponent(tabsSizeLarge).bind({});
+
+export default {
+  title: 'Components/Tabs',
+  args: {
+    appearance: 'transparent',
+    disabled: false,
+    orientation: 'horizontal',
+    size: 'medium',
+  },
+  argTypes: {
+    appearance: {
+      options: Object.values(TabsAppearanceValues),
+      defaultValue: 'transparent',
```

Comment:
Let's be sure (for all the instances here) we use the actual const object to enumerate the default value.